### DISCUSSION TITLE
建议增加原文和翻译颜色设置，增加阅读快感

### DISCUSSION THREAD

**Discussion by chutianshu1981** [immersive-translate/immersive-translate#1536] [Ideas]
### 新功能 | New Feature

1.可以自行设置原文、译文为不同颜色（或者字体）
2.可以自行设置原文、译文为不同底色（背景色）

### 为什么您想要添加这些功能 | Why you want to add these features

增加阅读时的体验度

### 提议 | Proposal

添加CSS

**Comment by cssmagic** [comment]
插件设置 > 基本设置 > 译文显示样式 > 自定义颜色和大小 & 设置字体

可以满足你的需求吗？

**Comment by theowenyoung** [comment]
为了减少不兼容行，沉浸式翻译的插件并不会对原文做任何标记，所以目前应该是无法操作原文。只能修改译文的样式。
